src/System.Net.WebSockets.Client/tests/ClientWebSocketTest.cs

```diff
@@ -465,6 +509,89 @@ public static IEnumerable<object[]> UnavailableWebSocketServers
         }
 
         [ConditionalTheory("WebSocketsSupported"), MemberData("EchoServers")]
+        public async Task CloseOutputAsync_ClientInitiated_CanReceive_CanClose(Uri server)
+        {
+            string message = "Hello WebSockets!";
+
+            using (ClientWebSocket cws = await WebSocketHelper.GetConnectedWebSocket(server, TimeOutMilliseconds, _output))
+            {
+                var cts = new CancellationTokenSource(TimeOutMilliseconds);
+
+                var closeStatus = WebSocketCloseStatus.InvalidPayloadData;
+                string closeDescription = "CloseOutputAsync_Client_InvalidPayloadData";
+
+                await cws.SendAsync(WebSocketData.GetBufferFromText(message), WebSocketMessageType.Text, true, cts.Token);
+                // Need a short delay as per WebSocket rfc6455 section 5.5.1 there isn't a requirement to receive any
+                // data fragments after a close has been sent. The delay allows the received data fragment to be
+                // available before calling close. The WinRT MessageWebSocket implementation doesn't allow receiving
+                // after a call to Close.
```

Comment:
/cc @mconnew 

This was added for the NetNative implementation only. It's not going to be a problem until we can run NetCore50 tests.
My proposal is to leave this as-is and follow-up if we see issues.

Internal NCL test passes showed no random failures on either flavors.